templates/es2015/server.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Module dependencies.
+ */
+
+import app from './app';
+import {default as debugModule} from 'debug';
```

Comment:
The default module isn't named debugModule, so you have to alias it to avoid a naming collision with debug which if I remember correctly lies elsewhere.